src/components/App/App.tsx

```diff
@@ -12,32 +12,76 @@ import React, { FC } from "react";
-                        isItalics={name.isItalics}
-                        onInputChange={updateName}
-                    />
-                    {/* <UsernameInput label={"Title"} />
-                    <UsernameInput label={"Company"} />
-                    <UsernameInput label={"Email"} />
-                    <UsernameInput label={"Phone number"} />
-                    <UsernameInput label={"LinkedIn"} /> */}
-                    <ImgInput />
-                    <TablePreview />
-                </Flex>
-            </Layout>
-        </ProviderV3>
+        <AuthenticationProvider>
+            <ProviderV3 theme={isDarkMode ? darkTheme : lightTheme} colorScheme={"light"} height="100%">
+                <ToastContainer />
+                <Layout>
+                    <Flex
+                        marginX="size-300"
+                        alignItems="start"
+                        direction="column"
+                        justifyContent="center"
+                        gap={"size-200"}
+                    >
+                        <UsernameInput
+                            type="name"
+                            label={"Name"}
+                            text={name.textValue}
+                            isBold={name.isBold}
+                            isItalics={name.isItalics}
+                            onInputChange={updateState}
+                        />
+                        <UsernameInput
+                            type="title"
+                            label={"Title"}
+                            text={title.textValue}
+                            isBold={title.isBold}
+                            isItalics={title.isItalics}
+                            onInputChange={updateState}
+                        />
+
+                        <UsernameInput
+                            type="company"
+                            label={"Company"}
+                            text={company.textValue}
+                            isBold={company.isBold}
+                            isItalics={company.isItalics}
+                            onInputChange={updateState}
+                        />
+                        <UsernameInput
+                            type="email"
+                            label={"Email"}
+                            text={email.textValue}
+                            isBold={email.isBold}
+                            isItalics={email.isItalics}
+                            onInputChange={updateState}
+                        />
+                        <UsernameInput
+                            type="phone"
+                            label={"Phone"}
+                            text={phone.textValue}
+                            isBold={phone.isBold}
+                            isItalics={phone.isItalics}
+                            onInputChange={updateState}
+                        />
+                        <UsernameInput
+                            type="linkedin"
```

Comment:
Shouldn't we be using `type` from the state object? i.e. `type={linkedin.type}`

Otherwise the type in the object and this type will not be synced

Reply:
🦅 👁️‍🗨️